### ISSUE TITLE
Unwanted CookieBasedSession in Nancy Implementation

### ISSUE THREAD

**Issue by devandy** [SimpleAuthentication/SimpleAuthentication#157]
In the Nancy implementation, there is a class that set the CookieBasedSession (EnableSessions.cs) on application start up.
This is bad because ovverides the application specific implementation of session management.
I think that that class has to be removed, and let the developer decide which type of session implementation is the best choice.

**Comment by Hihaj** [comment]
@PureKrome :+1:

**Comment by devandy** [comment]
Yes, I expect that if i don't enable session support I'll get an exeption, it's not a SA concern. Obviously I expect that there is a note in the SA documentation about Nancy Session Support requirement.

**Comment by PureKrome** [comment]
Sweet! I'll add a label here so I know to document this issue when I get around to releasing the vnext.

**Comment by devandy** [comment]
@PureKrome :+1:
